ts/features/walletV3/payment/components/WalletPaymentConfirmContent.tsx

```diff
@@ -89,7 +96,9 @@ export const WalletPaymentConfirmContent = ({
         iconName="psp"
       />
       <ModuleCheckout
-        ctaText={I18n.t("payment.confirm.editButton")}
+        ctaText={
```

Comment:
We should also remove the `onPress` if `pspList` is empty. If `pspList` is empty this module should not be interactable at all.

Reply:
Inside the PR https://github.com/pagopa/io-app-design-system/pull/185, there is a conditional rendering and as you can see if the `ModuleCheckout` component doesn't have the `ctaText` prop set, it shows only a simple `View` that is not interactable at all (this means that the `onPress` event, will not be fired)